sdk/quota/arm-quota/review/arm-quota-node.api.md

```diff
@@ -161,67 +150,56 @@ export interface GroupQuotaLimitProperties extends GroupQuotaDetails {
 }
 
 // @public
-export interface GroupQuotaLimits {
-    list(managementGroupId: string, groupQuotaName: string, resourceProviderName: string, location: string, options?: GroupQuotaLimitsListOptionalParams): Promise<GroupQuotaLimitsListResponse>;
+export interface GroupQuotaLimitsListOptionalParams extends OperationOptions {
 }
 
 // @public
-export interface GroupQuotaLimitsListOptionalParams extends coreClient.OperationOptions {
+export interface GroupQuotaLimitsOperations {
+    list: (managementGroupId: string, groupQuotaName: string, resourceProviderName: string, location: string, options?: GroupQuotaLimitsListOptionalParams) => Promise<GroupQuotaLimitList>;
 }
 
 // @public
-export type GroupQuotaLimitsListResponse = GroupQuotaLimitList;
-
-// @public
-export interface GroupQuotaLimitsRequest {
-    beginUpdate(managementGroupId: string, groupQuotaName: string, resourceProviderName: string, location: string, options?: GroupQuotaLimitsRequestUpdateOptionalParams): Promise<SimplePollerLike<OperationState<GroupQuotaLimitsRequestUpdateResponse>, GroupQuotaLimitsRequestUpdateResponse>>;
-    beginUpdateAndWait(managementGroupId: string, groupQuotaName: string, resourceProviderName: string, location: string, options?: GroupQuotaLimitsRequestUpdateOptionalParams): Promise<GroupQuotaLimitsRequestUpdateResponse>;
-    get(managementGroupId: string, groupQuotaName: string, requestId: string, options?: GroupQuotaLimitsRequestGetOptionalParams): Promise<GroupQuotaLimitsRequestGetResponse>;
-    list(managementGroupId: string, groupQuotaName: string, resourceProviderName: string, filter: string, options?: GroupQuotaLimitsRequestListOptionalParams): PagedAsyncIterableIterator<SubmittedResourceRequestStatus>;
+export interface GroupQuotaLimitsRequestGetOptionalParams extends OperationOptions {
 }
 
 // @public
-export interface GroupQuotaLimitsRequestGetOptionalParams extends coreClient.OperationOptions {
+export interface GroupQuotaLimitsRequestListOptionalParams extends OperationOptions {
 }
 
 // @public
-export type GroupQuotaLimitsRequestGetResponse = SubmittedResourceRequestStatus;
-
-// @public
-export interface GroupQuotaLimitsRequestListNextOptionalParams extends coreClient.OperationOptions {
+export interface GroupQuotaLimitsRequestOperations {
+    get: (managementGroupId: string, groupQuotaName: string, requestId: string, options?: GroupQuotaLimitsRequestGetOptionalParams) => Promise<SubmittedResourceRequestStatus>;
+    list: (managementGroupId: string, groupQuotaName: string, resourceProviderName: string, filter: string, options?: GroupQuotaLimitsRequestListOptionalParams) => PagedAsyncIterableIterator<SubmittedResourceRequestStatus>;
+    update: (managementGroupId: string, groupQuotaName: string, resourceProviderName: string, location: string, options?: GroupQuotaLimitsRequestUpdateOptionalParams) => PollerLike<OperationState<GroupQuotaLimitList>, GroupQuotaLimitList>;
 }
```

Comment:
Could you base on https://github.com/Azure/azure-rest-api-specs/pull/37584/files to re-generate to check if the parameter order would impact us?